TestResultSummaryService/parsers/Tap.js

```diff
@@ -1,13 +1,14 @@
 const Parser = require('./Parser');
 const Utils = require(`./Utils`);
-const decompress = require('decompress');
```

Comment:
Could we remove decompress from package.json if we do not use it?

Reply:
@nadeensami could you remove decompress in package.json? Thanks

Reply:
Removed decompress package and fixed API processing issues!